monai/metrics/utils.py

```
@@ -143,6 +165,7 @@ def get_mask_edges(
     crop: bool = True,
     spacing: Sequence | None = None,
     always_return_as_numpy: bool = False,
+    ignore_index: int | None = None,
```

Ruff (0.15.5)

[warning] Unused function argument: `ignore_index`

(ARG001)